dt-assets/js/comments.js

```diff
@@ -411,7 +411,7 @@ jQuery(document).ready(function($) {
     document.querySelectorAll('#comments-wrapper [data-toggle]').forEach((element) => {
       const dropdownId = $(element).data('toggle')
       const dropdownElement = document.querySelector(`#${dropdownId}`)
-      element.addEventListener('click', (e) => {
+      element.addEventListener('mouseover', (e) => {
```

Comment:
@prykon this behavior should be click.
One the add reaction button has been clicked, we want the user to be able to hover over the icon options and see a tooltip text.